presto-parser/src/main/java/com/facebook/presto/sql/parser/AstBuilder.java

```diff
@@ -1230,6 +1231,16 @@ public Node visitFunctionCall(SqlBaseParser.FunctionCallContext context)
 
             return new TryExpression(getLocation(context), (Expression) visit(getOnlyElement(context.expression())));
         }
+        if (name.toString().equalsIgnoreCase("$internal$bind")) {
```

Comment:
Presto uses AST to represent expressions during planning, and transmits optimized expressions (as AST) between coordinator and worker